packages/contracts-periphery/contracts/testing/helpers/Multicall3.sol

```diff
@@ -0,0 +1,261 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.15;
+
+// https://github.com/mds1/multicall/blob/master/src/Multicall3.sol
```

Comment:
Copied over contract from https://github.com/mds1/multicall/blob/master/src/Multicall3.sol

Reply:
Can you install this with forge or npm?

Reply:
I tried with forge install, but because forge uses git submodules, it gets added to the repo top-level .gitmodules, which isn't ideal. Do you have a suggest way of installing submodules scoped to the `contracts-periphery` package?

Reply:
It should be fine if its in the top level `gitmodules` - I believe the build always happens with the monorepo, you can still have the path its installed to being in `packages/contracts-periphery`. Will need to change CI to ensure the submodules are installed

Reply:
Updated the PR to install the repo as a submodule using `forge install`. This required using `deployCode` since the solidity versions aren't compatible (Multicall3.sol uses 0.8.12).